pmlc/conversion/linalg_to_pxa/passes.td

```diff
@@ -0,0 +1,20 @@
+#ifndef __PMLC_CONVERSION_LINALG_TO_PXA_PASSES__
+#define __PMLC_CONVERSION_LINALG_TO_PXA_PASSES__
+
+include "mlir/Pass/PassBase.td"
+
+def LowerLinalgToPXA : Pass<"convert-linalg-to-pxa", "mlir::ModuleOp"> {
+  let summary = "Convert Linalg dialect to PXA dialect";
+  let constructor = "pmlc::conversion::linalg_to_pxa::createLowerLinalgToPXAPass()";
+  let dependentDialects = [
+    "mlir::AffineDialect",
+    "mlir::linalg::LinalgDialect",
```

Comment:
Since this is the input dialect, I don't think we need to specify it in the dependent dialects.